scripts/quickstart.py

```diff
@@ -7,11 +7,13 @@
 import coiled
 import dask.dataframe as dd
 from dask.distributed import Client
+import pandas as pd
 
 SOFTWARE = os.environ["SOFTWARE_ENV"]
 
 cluster = coiled.Cluster(
     software=SOFTWARE,
+    name="github-action-", #can we add an identifier for the github action here?
```

Comment:
yes we can, what can of identifier you want, just a random number? 
This could be 
```
name=f"github-action-{uuid.uuid4().hex[:8]}"
```
we will need to import import uuid, if you want something more specific then it depends on what we want.

Reply:
It looks like `github.run_id ($GITHUB_RUN_ID)` is what you want. We will need to export it as an env variable in the action and get it in the script. Similar to what we do with the software environment.